README.md

```diff
@@ -29,7 +29,7 @@ Documentation
   the low-level file format of Teeworlds/DDNet demos (replays).
 - [huffman](doc/huffman.md). Homebrew compression format using [Huffman
   coding](https://en.wikipedia.org/wiki/Huffman_coding), used in demos and over
-  the network.
+  the network. Comes as a rust create but also has [python bindings](./uniffi/huffman/)
```

Comment:
This is the documentation section, so the link doesn't make sense here. There's an extra `./` in front.

There's already a link to the uniffi stuff below, so I don't think we need to add one.

Reply:
Well I did not find it the first time I was looking for it. I searched for huffman and python in the readme and it did not help. I don't think it is obvious that the python huffman bindings are located in the uniffi directory.